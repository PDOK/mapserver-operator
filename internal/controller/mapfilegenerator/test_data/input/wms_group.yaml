apiVersion: pdok.nl/v2beta1
kind: WMS
metadata:
  name: hwh-hydrografie-v2-0
  labels:
    dataset-owner: hwh
    dataset: hydrografie
    service-version: v2_0
    service-type: wms
  annotations:
    lifecycle-phase: prod
    service-bundle-id: d30bdf62-1c12-45a5-a57d-367e642ef118
spec:
  general:
    datasetOwner: hwh
    dataset: hydrografie
    serviceVersion: v2_0
  kubernetes:
    resources:
      limits:
        ephemeralStorage: 1544Mi
      requests:
        ephemeralStorage: 1544Mi
  service:
    title: Waterschappen Hydrografie (INSPIRE geharmoniseerd) WMS
    abstract:
      "Deze dataset is gebaseerd op (niet geharmoniseerde) data van alle waterschappen
      in Nederland conform INSPIRE. De dataset bevat de volgende INSPIRE objecten:
      CrossingLine, CrossingPoint, DamOrWeir, DrainageBasin, Embankment, Lock, Sluice,
      Watercourse."
    authority:
      name: hwh
      url: https://www.hetwaterschapshuis.nl/
    dataEPSG: EPSG:28992
    extent: -59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961
    inspire: true
    keywords:
      - Aquaduct
      - Brug
      - Duikersifonhevel
      - Stuw
      - Vastedam
      - AfvoerAanvoergebied
      - Waterkering
      - Sluis
      - HydroObject
      - HVD
      - Aardobservatie en milieu
    metadataIdentifier: 871a58f8-c9f1-41a4-be37-0f059e0f886f
    stylingAssets:
      configMapRefs:
        - name: includes
          keys:
            - bridge.symbol
            - x.symbol
      blobKeys:
        - resources/images/hwh/hydrografie/bridge.png
    layers:
      - abstract: Watercourse
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Waterbody.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - delineationknown
              - endlifespanversion
              - length
              - level
              - localid
              - localtype
              - name
              - namespace
              - origin
              - persistence
              - streamorder
              - tidal
              - widthrange
            geometryType: MultiLineString
            table: watercourse
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        keywords:
          - Hydroobject
          - Waterbody
          - Watercourse
          - River
          - Stream
          - Lake
          - Reservoir
        name: HY.PhysicalWaters.Waterbodies
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.Waterbodies.Default
            title: Water bodies default style
            visualization: watercourse.style
        title: Waterbody
        maxScale: 50000.0
        minScale: 1.0
        visible: true
      - abstract: Drainage basin
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Catchment.gpkg
            columns:
              - gml_id
              - area
              - beginlifespanversion
              - endlifespanversion
              - localid
              - name
              - namespace
              - order
              - orderscheme
              - origin
              - outlet
              - scope
            geometryType: MultiPolygon
            table: drainagebasin
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        keywords:
          - AfvoergebiedAanvoergebied
          - Catchment
          - Basin
          - Catchment Area
          - Drainage basin
        name: HY.PhysicalWaters.Catchments
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.Catchments.Default
            title: Drainage Basin default style
            visualization: drainagebasin.style
        title: Catchment
        maxScale: 50000.0
        minScale: 1.0
        visible: true
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis
          - Bridge
          - Aquaduct
          - Dam
          - Weir
          - Lock
          - Ford
          - Dyke
        maxScale: 50000.0
        minScale: 1.0
        name: HY.PhysicalWaters.ManMadeObject
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Man-made objects default style
        title: Man-made Object
        visible: true
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Man-madeObject.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - endlifespanversion
              - localid
              - name
              - namespace
            geometryType: Point
            table: lock
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        group: HY.PhysicalWaters.ManMadeObject
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis.
        name: HY.PhysicalWaters.ManMadeObject.lock
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Lock
            visualization: lock.style
        title: Man-made Object
        maxScale: 50000.0
        minScale: 1.0
        visible: false
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Man-madeObject.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - endlifespanversion
              - localid
              - name
              - namespace
              - type
            geometryType: Point
            table: crossingpoint
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        group: HY.PhysicalWaters.ManMadeObject
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis.
        name: HY.PhysicalWaters.ManMadeObject.crossingpoint
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Crossing Point
            visualization: crossingpoint.style
        title: Man-made Object
        maxScale: 50000.0
        minScale: 1.0
        visible: false
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Man-madeObject.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - endlifespanversion
              - localid
              - name
              - namespace
            geometryType: Point
            table: sluice
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        group: HY.PhysicalWaters.ManMadeObject
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis.
        name: HY.PhysicalWaters.ManMadeObject.sluice
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Sluice
            visualization: sluice.style
        title: Man-made Object
        maxScale: 50000.0
        minScale: 1.0
        visible: false
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Man-madeObject.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - endlifespanversion
              - localid
              - name
              - namespace
            geometryType: MultiLineString
            table: embankment
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        group: HY.PhysicalWaters.ManMadeObject
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis.
        name: HY.PhysicalWaters.ManMadeObject.embankment
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Embankment
            visualization: embankment.style
        title: Man-made Object
        maxScale: 50000.0
        minScale: 1.0
        visible: false
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Man-madeObject.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - endlifespanversion
              - localid
              - name
              - namespace
              - type
            geometryType: MultiLineString
            table: crossingline
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        group: HY.PhysicalWaters.ManMadeObject
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis.
        name: HY.PhysicalWaters.ManMadeObject.crossingline
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Crossing Line
            visualization: crossingline.style
        title: Man-made Object
        maxScale: 50000.0
        minScale: 1.0
        visible: false
      - abstract: Crossing line, crossing point, dam or weir, sluice, embankment, lock.
        data:
          gpkg:
            blobKey: geopackages/hwh/hydrografie/aaa/1/Man-madeObject.gpkg
            columns:
              - gml_id
              - beginlifespanversion
              - condition
              - endlifespanversion
              - localid
              - name
              - namespace
            geometryType: Point
            table: damorweir
        datasetMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        group: HY.PhysicalWaters.ManMadeObject
        keywords:
          - Man Made
          - Duikersifonhevel
          - Aquaduct
          - Brug
          - Stuw
          - Vastedam
          - Waterkering
          - Sluis.
        name: HY.PhysicalWaters.ManMadeObject.damorweir
        sourceMetadataIdentifier: 07575774-57a1-4419-bab4-6c88fdeb02b2
        styles:
          - name: HY.PhysicalWaters.ManMadeObject.Default
            title: Dam or Weir
            visualization: damorweir.style
        title: Man-made Object
        maxScale: 50000.0
        minScale: 1.0
        visible: false
