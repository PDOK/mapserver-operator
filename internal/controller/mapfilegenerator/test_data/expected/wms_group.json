{
    "authority_url": "https://www.hetwaterschapshuis.nl/",
    "automatic_casing": true,
    "data_epsg": "EPSG:28992",
    "dataset_owner": "hwh",
    "epsg_list": [
        "EPSG:28992",
        "EPSG:25831",
        "EPSG:25832",
        "EPSG:3034",
        "EPSG:3035",
        "EPSG:3857",
        "EPSG:4258",
        "EPSG:4326",
        "CRS:84"
    ],
    "group_layers": [
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "name": "HY.PhysicalWaters.ManMadeObject",
            "style_name": "HY.PhysicalWaters.ManMadeObject.Default",
            "style_title": "Man-made objects default style",
            "title": "Man-made Object"
        }
    ],
    "layers": [
        {
            "abstract": "Watercourse",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "delineationknown"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "length"
                },
                {
                    "name": "level"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "localtype"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                },
                {
                    "name": "origin"
                },
                {
                    "name": "persistence"
                },
                {
                    "name": "streamorder"
                },
                {
                    "name": "tidal"
                },
                {
                    "name": "widthrange"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "MultiLineString",
            "gpkg_path": "/srv/data/gpkg/Waterbody.gpkg",
            "keywords": "Hydroobject,Waterbody,Watercourse,River,Stream,Lake,Reservoir",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.Waterbodies",
            "styles": [
                {
                    "path": "/styling/watercourse.style",
                    "title": "Water bodies default style"
                }
            ],
            "tablename": "watercourse",
            "title": "Waterbody"
        },
        {
            "abstract": "Drainage basin",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "area"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                },
                {
                    "name": "order"
                },
                {
                    "name": "orderscheme"
                },
                {
                    "name": "origin"
                },
                {
                    "name": "outlet"
                },
                {
                    "name": "scope"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "MultiPolygon",
            "gpkg_path": "/srv/data/gpkg/Catchment.gpkg",
            "keywords": "AfvoergebiedAanvoergebied,Catchment,Basin,Catchment Area,Drainage basin",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.Catchments",
            "styles": [
                {
                    "path": "/styling/drainagebasin.style",
                    "title": "Drainage Basin default style"
                }
            ],
            "tablename": "drainagebasin",
            "title": "Catchment"
        },
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "Point",
            "gpkg_path": "/srv/data/gpkg/Man-madeObject.gpkg",
            "group_name": "HY.PhysicalWaters.ManMadeObject",
            "keywords": "Man Made,Duikersifonhevel,Aquaduct,Brug,Stuw,Vastedam,Waterkering,Sluis.",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.ManMadeObject.lock",
            "styles": [
                {
                    "path": "/styling/lock.style",
                    "title": "Lock"
                }
            ],
            "tablename": "lock",
            "title": "Man-made Object"
        },
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                },
                {
                    "name": "type"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "Point",
            "gpkg_path": "/srv/data/gpkg/Man-madeObject.gpkg",
            "group_name": "HY.PhysicalWaters.ManMadeObject",
            "keywords": "Man Made,Duikersifonhevel,Aquaduct,Brug,Stuw,Vastedam,Waterkering,Sluis.",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.ManMadeObject.crossingpoint",
            "styles": [
                {
                    "path": "/styling/crossingpoint.style",
                    "title": "Crossing Point"
                }
            ],
            "tablename": "crossingpoint",
            "title": "Man-made Object"
        },
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "Point",
            "gpkg_path": "/srv/data/gpkg/Man-madeObject.gpkg",
            "group_name": "HY.PhysicalWaters.ManMadeObject",
            "keywords": "Man Made,Duikersifonhevel,Aquaduct,Brug,Stuw,Vastedam,Waterkering,Sluis.",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.ManMadeObject.sluice",
            "styles": [
                {
                    "path": "/styling/sluice.style",
                    "title": "Sluice"
                }
            ],
            "tablename": "sluice",
            "title": "Man-made Object"
        },
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "MultiLineString",
            "gpkg_path": "/srv/data/gpkg/Man-madeObject.gpkg",
            "group_name": "HY.PhysicalWaters.ManMadeObject",
            "keywords": "Man Made,Duikersifonhevel,Aquaduct,Brug,Stuw,Vastedam,Waterkering,Sluis.",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.ManMadeObject.embankment",
            "styles": [
                {
                    "path": "/styling/embankment.style",
                    "title": "Embankment"
                }
            ],
            "tablename": "embankment",
            "title": "Man-made Object"
        },
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                },
                {
                    "name": "type"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "MultiLineString",
            "gpkg_path": "/srv/data/gpkg/Man-madeObject.gpkg",
            "group_name": "HY.PhysicalWaters.ManMadeObject",
            "keywords": "Man Made,Duikersifonhevel,Aquaduct,Brug,Stuw,Vastedam,Waterkering,Sluis.",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.ManMadeObject.crossingline",
            "styles": [
                {
                    "path": "/styling/crossingline.style",
                    "title": "Crossing Line"
                }
            ],
            "tablename": "crossingline",
            "title": "Man-made Object"
        },
        {
            "abstract": "Crossing line, crossing point, dam or weir, sluice, embankment, lock.",
            "columns": [
                {
                    "name": "fuuid"
                },
                {
                    "name": "gml_id"
                },
                {
                    "name": "beginlifespanversion"
                },
                {
                    "name": "condition"
                },
                {
                    "name": "endlifespanversion"
                },
                {
                    "name": "localid"
                },
                {
                    "name": "name"
                },
                {
                    "name": "namespace"
                }
            ],
            "dataset_metadata_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "dataset_source_id": "07575774-57a1-4419-bab4-6c88fdeb02b2",
            "geometry_type": "Point",
            "gpkg_path": "/srv/data/gpkg/Man-madeObject.gpkg",
            "group_name": "HY.PhysicalWaters.ManMadeObject",
            "keywords": "Man Made,Duikersifonhevel,Aquaduct,Brug,Stuw,Vastedam,Waterkering,Sluis.",
            "layer_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
            "maxscale": "50000",
            "minscale": "1",
            "name": "HY.PhysicalWaters.ManMadeObject.damorweir",
            "styles": [
                {
                    "path": "/styling/damorweir.style",
                    "title": "Dam or Weir"
                }
            ],
            "tablename": "damorweir",
            "title": "Man-made Object"
        }
    ],
    "maxSize": "4000",
    "outputformat_jpg": "jpg",
    "outputformat_png8": "png",
    "service_abstract": "Deze dataset is gebaseerd op (niet geharmoniseerde) data van alle waterschappen in Nederland conform INSPIRE. De dataset bevat de volgende INSPIRE objecten: CrossingLine, CrossingPoint, DamOrWeir, DrainageBasin, Embankment, Lock, Sluice, Watercourse.",
    "service_accessconstraints": "https://creativecommons.org/publicdomain/zero/1.0/deed.nl",
    "service_extent": "-59188.44333693248 304984.64144318487 308126.88473339565 858328.516489961",
    "service_keywords": "Aquaduct,Brug,Duikersifonhevel,Stuw,Vastedam,AfvoerAanvoergebied,Waterkering,Sluis,HydroObject,HVD,Aardobservatie en milieu,infoMapAccessService",
    "service_metadata_id": "871a58f8-c9f1-41a4-be37-0f059e0f886f",
    "service_namespace_prefix": "hydrografie",
    "service_namespace_uri": "http://hydrografie.geonovum.nl",
    "service_onlineresource": "https://service.pdok.nl",
    "service_path": "/hwh/hydrografie/wms/v2_0",
    "service_title": "Waterschappen Hydrografie (INSPIRE geharmoniseerd) WMS",
    "symbols": [
        "/styling/bridge.symbol",
        "/styling/x.symbol"
    ],
    "templates": "/srv/data/config/templates"
}